spyder/plugins/completion/api.py

```diff
@@ -1316,7 +1316,7 @@ def get_action(self, name, context=None, plugin=None):
         """
         return self.main.get_action(name, context=context, plugin=plugin)
 
-    def create_application_menu(self, menu_id, title, dynamic=True):
+    def create_application_menu(self, menu_id, title):
```

Comment:
The `dynamic` kwarg is no longer necessary for main menus because all of them are now migrated to the new API. So, I removed it in this PR.